mailpoet/assets/css/src/components-automation-editor/_panel.scss

```diff
@@ -57,3 +57,73 @@
   color: #757575;
   font-style: italic;
 }
+
```

Comment:
Are these styles taken from the Gutenberg repo? We could add a permalink above, for instance like [here](https://github.com/mailpoet/mailpoet/blob/3ea730a0b642b6c59875f0a0fec4e858d71315a7/mailpoet/assets/css/src/components-automation-editor/_step-card.scss#L1).

Reply:
The naming is a bit different and sometimes the selectors/selector-cascading. I wonder, should I align it more?